<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2e2e2e;
}
QWidget {
    background-color: #2e2e2e;
}
QLabel {
    color: #ffffff;
    font-family: 'Roboto', sans-serif;
    font-size: 25px;
}
QPushButton {
    background-color: #444444;
    color: #e0e0e0;
    border: 1px solid #333333;
    border-radius: 5px;
    font-family: 'Roboto', sans-serif;
    font-size: 18px;
}
QPushButton:hover {
    background-color: #555555;
}
QPushButton:pressed {
    background-color: #666666;
}
QPushButton:disabled {
    background-color: #2e2e2e;
    color: #8a8a8a;
}

/* Przycisk liczb */
QPushButton#pushButton_0,
QPushButton#pushButton_1,
QPushButton#pushButton_2,
QPushButton#pushButton_3,
QPushButton#pushButton_4,
QPushButton#pushButton_5,
QPushButton#pushButton_6,
QPushButton#pushButton_7,
QPushButton#pushButton_8,
QPushButton#pushButton_9 {
    background-color: #3a3a3a;
}
QPushButton#pushButton_0:hover,
QPushButton#pushButton_1:hover,
QPushButton#pushButton_2:hover,
QPushButton#pushButton_3:hover,
QPushButton#pushButton_4:hover,
QPushButton#pushButton_5:hover,
QPushButton#pushButton_6:hover,
QPushButton#pushButton_7:hover,
QPushButton#pushButton_8:hover,
QPushButton#pushButton_9:hover {
    background-color: #4a4a4a;
}
QPushButton#pushButton_0:pressed,
QPushButton#pushButton_1:pressed,
QPushButton#pushButton_2:pressed,
QPushButton#pushButton_3:pressed,
QPushButton#pushButton_4:pressed,
QPushButton#pushButton_5:pressed,
QPushButton#pushButton_6:pressed,
QPushButton#pushButton_7:pressed,
QPushButton#pushButton_8:pressed,
QPushButton#pushButton_9:pressed {
    background-color: #5a5a5a;
}

/* Przycisk CLEAR */
QPushButton#pushButton_clear {
    background-color: #5a5a5a;
}
QPushButton#pushButton_clear:hover {
    background-color: #6a6a6a;
}
QPushButton#pushButton_clear:pressed {
    background-color: #7a7a7a;
}

/* Przycisk wynikowy */
QPushButton#pushButton_result {
    background-color: #4e5f4e;
}
QPushButton#pushButton_result:hover {
    background-color: #5e6f5e;
}
QPushButton#pushButton_result:pressed {
    background-color: #6e7f6e;
}

/* Przycisk BACKSPACE */
QPushButton#pushButton_backspace {
    background-color: #5e4f4f;
    color: #ffffff;
}
QPushButton#pushButton_backspace:hover {
    background-color: #6e5f5f;
}
QPushButton#pushButton_backspace:pressed {
    background-color: #4e3f3f;
}

/* Operacje matematyczne */
QPushButton#pushButton_addition, 
QPushButton#pushButton_substract, 
QPushButton#pushButton_multiply, 
QPushButton#pushButton_division,
QPushButton#pushButton_percent,
QPushButton#pushButton_absolute, 
QPushButton#pushButton_power, 
QPushButton#pushButton_root,
QPushButton#pushButton_modulo,
QPushButton#pushButton_log,
QPushButton#pushButton_ln,
QPushButton#pushButton_euler,
QPushButton#pushButton_pi,
QPushButton#pushButton_factorial   {
    background-color: #484848;
}
QPushButton#pushButton_addition:hover,
QPushButton#pushButton_substract:hover,
QPushButton#pushButton_multiply:hover,
QPushButton#pushButton_division:hover,
QPushButton#pushButton_percent:hover,
QPushButton#pushButton_absolute:hover,
QPushButton#pushButton_power:hover,
QPushButton#pushButton_root:hover,
QPushButton#pushButton_modulo:hover,
QPushButton#pushButton_log:hover,
QPushButton#pushButton_ln:hover,
QPushButton#pushButton_eule:hover,
QPushButton#pushButton_pi:hover,
QPushButton#pushButton_factorial:hover  {
    background-color: #585858;
}
QPushButton#pushButton_addition:pressed,
QPushButton#pushButton_substract:pressed,
QPushButton#pushButton_multiply:pressed,
QPushButton#pushButton_division:pressed,
QPushButton#pushButton_percent:pressed,
QPushButton#pushButton_absolute:pressed,
QPushButton#pushButton_power:pressed,
QPushButton#pushButton_root:pressed,
QPushButton#pushButton_modulo:pressed,
QPushButton#pushButton_log:pressed,
QPushButton#pushButton_ln:pressed,
QPushButton#pushButton_euler:pressed,
QPushButton#pushButton_pi:pressed,
QPushButton#pushButton_factorial:pressed  {
    background-color: #686868;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_substract">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addition">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_inverse">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_power">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>x^n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_root">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>โx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backspace">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>โซ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_absolute">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>|x|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_euler">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pi">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ฯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_factorial">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>n!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
